apps/scan/backend/src/electrical_testing/task_controller.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, test } from 'vitest';
+import { TaskController } from './task_controller';
+
+describe('TaskController', () => {
+  test('is initially not running', () => {
+    const loop = new TaskController();
```

Comment:
nit: Let's switch from `loop` to `task` here like we did in the application code for consistency